api/jobs_test.go

```diff
@@ -155,3 +156,141 @@ func TestCreateJobHandlerWithInvalidID(t *testing.T) {
 		})
 	})
 }
+
+func TestGetJobsHandler(t *testing.T) {
+
```

Comment:
I see a lot of blank lines at the beginning of functions littered throughout all the code but it's completely inconsistent.. I think it's a lot better to choose one formatting guideline and stick to it. I also think preferably the blank line should be dropped, as encouraged by gofmt/golint. After a while it _will_ stop looking awkward to your eyes, honest :)

Reply:
It's a fair point. I have been using the gofmt tool but I guess it's leaving me to decide about the blank lines. I'm happy to remove as many as I can find.

Reply:
done